docs/src/network_analysis/crnt.md

```diff
@@ -1,219 +1,26 @@
-# [Network Analysis in Catalyst](@id network_analysis)
+# [Chemical Reaction Network Theory](@id network_analysis_structural_aspects)
```

Comment:
I'd probably call the file "crn_theory.md", more descriptive at a first glance for people unfamiliar with it.
(but feel free to keep current name as well!)